bootstrap

```diff
@@ -49,6 +49,12 @@ main(Args) ->
                           false -> undefined
                       end,
 
+    %% Ensure, that the ebin folder exists after "make clean"
+    ok = filelib:ensure_dir("ebin/"),
+
+    %% Copy rebar.app into ebin
+    {ok, _} = file:copy("rebar.app", "ebin/rebar.app"),
```

Comment:
maybe we can add something like e.g. ?
```
    %% CouchDB's `make clean` clobbers all ebin directories, so protect rebar.app by
    %%  moving it out of harm's way, and copying it into ebin before building.
```

Reply:
Added your suggestion and added ebin/ to .gitignore.